app/api/tickets.py

```diff
@@ -31,6 +33,9 @@ def before_post(self, args, kwargs, data):
             raise ObjectNotFound({'parameter': 'event_id'},
                                  "Event: {} not found".format(data['event_id']))
 
+        if get_count(db.session.query(Ticket.id).filter_by(name=data['name'], event_id=int(data['event']))):
```

Comment:
This should be get_count(...) > 0 ?

Reply:
In case `get_count` return `0`, the if condition fails and the body does not get executed. So why is this change needed ?

Reply:
If it returns 0, it means there is no ticket of this name, so error should not be raised

Reply:
Error is geting raised only when `get_count is > 0`. In case of 0 error is not getting raised.